contents/blog/hacker-news-premortem.md

```diff
@@ -0,0 +1,152 @@
+---
+date: 2022-06-29
+title: The magic of a Hacker News Pre-Mortem
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+categories: ["CEO diaries", "Inside PostHog"]
+author: ["james-hawkins"]
+featuredImage: ../images/blog/ceo-diary.png
+featuredImageType: full
+---
+
+Imagine you're working on something for other developers that you really, _really_ want to be great.
+
+You're creating a [new startup](https://www.youtube.com/watch?v=KdtPNRzuKrk), you're [creating a blog post](https://www.youtube.com/watch?v=fGT5kFMU048), you're working on [a side project](https://www.youtube.com/watch?v=ab47XHidvwQ).
```

Comment:
```suggestion
You're creating a new startup, you're creating a blog post, you're working on a side project.
```

I really don't think those links add anything. People will just get distracted / annnoyed that they don't add any value.